docs/versions/content-patterns.md

```diff
@@ -0,0 +1,313 @@
+
+Consider breaking up procedures into sets of procedures if only one section differs between contexts.
+
+### Example
+
+To create a space: 
+
+:::::{tab-set}
+:group: stack-serverless
+
+::::{tab-item} Serverless
+:sync: serverless
+
+1. Click **Create space** or select the space you want to edit.
+2. Provide:
+
+    * A meaningful name and description for the space.
+    * A URL identifier. The URL identifier is a short text string that becomes part of the Kibana URL. Kibana suggests a URL identifier based on the name of your space, but you can customize the identifier to your liking. You cannot change the space identifier later.
+
+3. Customize the avatar of the space to your liking.
+4. Save the space.
+::::
+
+::::{tab-item} Stack v9
+:sync: stack
+
+1. Select **Create space** and provide a name, description, and URL identifier.
+   The URL identifier is a short text string that becomes part of the Kibana URL when you are inside that space. Kibana suggests a URL identifier based on the name of your space, but you can customize the identifier to your liking. You cannot change the space identifier once you create the space.
+
+2. Select a **Solution view**. This setting controls the navigation that all users of the space will get:
+   * **Search**: A light navigation menu focused on analytics and Search use cases. Features specific to Observability and Security are hidden.
+   * **Observability**: A light navigation menu focused on analytics and Observability use cases. Features specific to Search and Security are hidden.
+   * **Security**: A light navigation menu focused on analytics and Security use cases. Features specific to Observability and Search are hidden.
+   * **Classic**: All features from all solutions are visible by default using the classic, multilayered navigation menus. You can customize which features are visible individually.
+
+3. If you selected the **Classic** solution view, you can customize the **Feature visibility** as you need it to be for that space.
+
+   :::{note} 
+   Even when disabled in this menu, some Management features can remain visible to some users depending on their privileges. Additionally, controlling feature visibility is not a security feature. To secure access to specific features on a per-user basis, you must configure Kibana Security.
+   :::
+4. Customize the avatar of the space to your liking.
+5. Save your new space by selecting **Create space**.
+::::
+
+:::::
+
+You can edit all of the space settings you just defined at any time, except for the URL identifier.
+
+## Sibling bullets 
+
+**Use case:** Requirements, limits, other simple, mirrored facts.
+
+**Number to use:** Ideal is one per option (e.g. one per deployment type). You might consider nested bullets in limited cases.
+
+### Examples
+
+::::{tab-set}
+:group: one-two
+
+:::{tab-item} One
+:sync: one
+
+#### Required permissions
+
+* **Serverless**: `Admin` or equivalent
+* **Stack v9**: `kibana_admin` or equivalent
```

Comment:
Ditto my previous suggestion:
```suggestion
* **All Elastic Stack**: `kibana_admin` or equivalent
```

Reply:
@florent-leborgne I think it's safe to assume that we will have this "these docs are for v9 onwards" messaging somewhere. I think at minimum we need to build a page to explain these annotations, but I think we will also get a banner. let me check in with kaarina.